plugin-template

```diff
@@ -67,7 +67,7 @@ DEFAULT_SETTINGS = {
     "pulp_settings": None,
     "pulp_settings_s3": None,
     "pydocstyle": True,
-    "python_version": "3.8",
+    "python_version": "3.9",
```

Comment:
This is what we use in `setup_python` so all the CI is running on 3.9 by default?
And the python version in the container is determined by the choice of the base image, right?

Reply:
Yes x2, afaik

Reply:
To clarify my other comment:
I would move this line to be hard coded within this file away from the DEFAULT_SETTINGS (keep it easy to parameterize, but don't let the template_config decide). You can easily inject it around line 418.